src/herbivoresAndCarnivores.test.js

```diff
@@ -34,8 +34,7 @@ describe('Animal', () => {
 
       expect(carn)
         .toHaveProperty('health', 100);
-    }
-  );
+    });
 
   it(`Herbivore should have 'hide' method which inverts 'hidden' property`,
```

Comment:
The test description says the 'hide' method should invert the 'hidden' property, but according to the task checklist, the 'hide' method should only set the 'hidden' property to true and should not toggle it.